tools/seq_filter_by_id/seq_filter_by_id.xml

```diff
@@ -1,8 +1,7 @@
-<tool id="seq_filter_by_id" name="Filter sequences by ID" version="0.2.3">
+<tool id="seq_filter_by_id" name="Filter sequences by ID" version="0.2.5">
     <description>from a tabular file</description>
     <requirements>
         <requirement type="package" version="1.64">biopython</requirement>
-        <requirement type="python-module">Bio</requirement>
```

Comment:
OK then - I also suspected this was merely given as an example in the old documentation but never had any function.